src/ui/pages/Onboarding/Onboarding.tsx

```diff
@@ -22,47 +20,7 @@ const Onboarding = (): JSX.Element => {
 
         <div className="text-base">To continue, please setup a password</div>
 
-        <div className="py-4 w-full">
-          <Input
-            autoFocus
-            className="mb-4"
-            errorMessage={errors.password}
-            icon={
-              <Tooltip
-                className="info-tooltip"
-                title={
-                  <div>
-                    <p>Password requirements:</p>
-
-                    <p>- At least 8 characters</p>
-
-                    <p>- At least 1 upper case and letter</p>
-
-                    <p>- At least 1 lower case letter</p>
-
-                    <p>- At least 1 special character (!@#$%^&*)</p>
-
-                    <p>- At least 1 number</p>
-                  </div>
-                }
-              >
-                <Icon className="info-icon" fontAwesome="fa-info" />
-              </Tooltip>
-            }
-            id="password"
-            label="Password"
-            type="password"
-            {...register("password")}
-          />
-
-          <Input
-            errorMessage={errors.confirmPassword}
-            id="confirmPassword"
-            label="Confirm Password"
-            type="password"
-            {...register("confirmPassword")}
-          />
-        </div>
+        <PasswordInput isShowConfirmPassword isShowInfo errors={errors} register={register} />
```

Comment:
And here we need to have two separate `PasswordInput` components. One for `id="password"` and second for `id="confirmPassword"`